draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -475,7 +475,51 @@ TBD: Verifier provided `nonce`.
 
 # Privacy Considerations {#privacy-considerations}
 
-TBD: Holder provided nonce via `jti`.
+The Privacy Considerations in the SD-JWT specification
+[@!I-D.ietf-oauth-selective-disclosure-jwt] apply to this specification.
+Additionally, the following privacy considerations need to be taken into
+account when using SD-JWT VCs.
+
+## Unlinkability
+
+The Privacy Considerations in Section 10.5 in [@!I-D.ietf-oauth-selective-disclosure-jwt]
+apply especially to the `cnf` claim.
+
+## Type Identifier
+
+Issuers and Holders have to be aware that while this specification supports selective
+disclosure of claims of a given SD-JWT VC, the `type` claim is not selectively discloseable.
+In certain situations this could lead to unwanted leakage of additional context information.
+
+In general, Issuers are advised to choose `type` values following data minimation principles.
+For example, an Issuer issuing a SD-JWT VC to their citizens to enable them to prove their age,
+have to avoid putting information into the `type` value that allows third-parties
+to infer additional personal information about the Holder, e.g., country of residency, citizenship
+etc. A better approach is to have more generalized types, for example an age-proof SD-JWT VC type.
```

Comment:
I'm not sure if this can be our general recommendation. It would mean that a digital citizenship credential can never be used for age verification. It would be better to argue that a certain level of leakage may be acceptable (e.g., country of residence), but other details should never leak.

Reply:
Well, this means that an issuer should not call their credential type "CitizenOfUtopiaCredential" and instead just call it "CitizenCredential".